testing/trino-tests/src/test/java/io/trino/execution/TestEventListenerBasic.java

```diff
@@ -332,7 +333,7 @@ private void assertFailedQuery(Session session, @Language("SQL") String sql, Str
     public void testReferencedTablesAndRoutines()
             throws Exception
     {
-        QueryEvents queryEvents = runQueryAndWaitForEvents("SELECT sum(linenumber) FROM lineitem").getQueryEvents();
+        QueryEvents queryEvents = runQueryAndWaitForEvents("SELECT sum(linenumber) FROM lineitem WHERE floor(linenumber) = 3").getQueryEvents();
```

Comment:
Maybe add `HAVING sum(linenumber) > 1` to make sure the same function would be correctly reported twice with a different `clauseInfo`.

Reply:
Done